### ISSUE TITLE
I got 404 not found when use the docker project

### ISSUE THREAD

**Issue by sunshineo** [pipeviz/pipeviz#246]
https://github.com/pipeviz/docker-pipeviz
INFO[2015-12-20T08:17:45Z] Beginning request processing                  remote=192.168.99.1:64400 system=webapp uri=/ method=GET
INFO[2015-12-20T08:17:45Z] Request processing complete                   wall=48.269µs status=404 uri=/ system=webapp method=GET remote=192.168.99.1:64400

**Comment by sdboyer** [comment]
Yeah, sorry, we've let the docker images drift out of date a bit. I'll bring it up on our call tomorrow, so we can at least try to figure out what steps we should take (docs, getting rid of the docker files until we're more prepared to support them, etc.). Will update here once I have more info.

**Comment by eliza411** [comment]
Building the Docker image fails on: 

glide -q -y glide.yaml up
make: glide: Command not found
Makefile:6: recipe for target 'deps' failed

**Comment by eliza411** [comment]
I'll take a look at this after the holiday. It seems like glide makes assumptions about its environment that I need a bit longer to sort out.

**Comment by sunshineo** [comment]
I believe I fixed it
https://github.com/pipeviz/docker-pipeviz/pull/3
